### ISSUE TITLE
Misnamed files

### ISSUE THREAD

**Issue by EdPhere** [porteux/porteux#1]
The ISO files aren't bootable and should be renamed ZIP files.  They contain files and directories to be copied to a media to be made bootable.

**Comment by porteux** [comment]
@EdPhere, the ISOs are not zip files and they're bootable via VirtualBox or an ISO installer application like Rufus or Etcher.

What exactly are you trying to do?

**Comment by EdPhere** [comment]
I like to test the app or system to see if I like it before I invest hardware to keep it.  I normally down load ISOs and boot them using grub.   I've been able to boot Porteus that way since 3.0.  If I have to use 7zip to copy the files to a drive to boot it the ISO may as well be a zip file. 

I've booted ISOs in qemu in the past but booting takes forever.  It's something one starts before going to bed.

So, rather than get peoples expectations up that they can down load the file and boot it, rename it as a zip.  That way they know to test it requires some effort.
